vendor/k8s.io/kubernetes/pkg/api/validation/validation.go

```diff
@@ -2544,6 +2544,17 @@ func ValidateLimitRange(limitRange *api.LimitRange) field.ErrorList {
 			}
 		}
 
+		if limit.Type == api.LimitTypePersistentVolumeClaim {
+			_, minQuantityFound := limit.Min[api.ResourceStorage]
```

Comment:
Allowing either to be empty opens the limit range to whatever the underlying infrastructure wants to enforce. IDK if that's good or bad, but AWS, for example, would enforce a 1Gi minimum and some large max Gi.

0 is an effective "no min".